### PR TITLE
[PIM-6495] PEF data

### PR DESCRIPTION
**Pull request by pierallard**
- PIM-6497: Move "status" in the left panel
- PIM-6499: Add product completeness in the PEF header

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: frontend/require-context.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ define([], function() {
     return function(moduleName) {
         var modulePath = __contextPaths[moduleName];
         if (undefined === modulePath) {
-            console.error('Module "' + moduleName + '" not found. Please check youre requirejs.yml files.');
+            console.error('Module "' + moduleName + '" not found. Please check your requirejs.yml files.');
```
Comment (phaseinducer) [inline frontend/require-context.js]: :water_buffalo:

Comment (pierallard) [inline frontend/require-context.js]: Your fired @phaseinducer

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/meta/status-switcher.js [hunk 2]
```diff
@@ -43,14 +45,12 @@ define(
...
+                    label: __('pim_enrich.entity.product.meta.status'),
+                    enabledLabel: __('pim_enrich.entity.product.btn.enabled'),
+                    disabledLabel: __('pim_enrich.entity.product.btn.disabled')
                 }));
-                this.$el
-                    .addClass(status ? 'AknDropdownButton--apply' : 'AknDropdownButton--important')
-                    .removeClass(status ? 'AknDropdownButton--important' : 'AknDropdownButton--apply')
-                    .find('.AknCaret')
-                    .addClass(status ? 'AknCaret--apply' : 'AknCaret--important')
-                    .removeClass(status ? 'AknCaret--important' : 'AknCaret--apply');
```
Comment (tamarasaurus) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/meta/status-switcher.js]: 🔥 🔥 🔥

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/form/product-completeness.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="AknBadge AknBadge--big <%- completeness > 0 ? (completeness >= 100 ? 'AknBadge--enabled' : 'AknBadge--warning') : 'AknBadge--important' %>">
```
Comment (tamarasaurus) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/form/product-completeness.html]: can you do this completeness logic in the view js ?

**Pull request merged**